### PR TITLE
Update fallbacks for invalid zone data and some other minor tweaks

### PR DESCRIPTION
**Pull request by TeoTwawki**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

1. updates `!up` to work on any entity type and specify any distance, and [adds its counterpart](https://user-images.githubusercontent.com/6871475/168452799-80a6f3a3-e933-4fd1-b692-d61985c00f7e.png) `!down`
2. add some notes for future work/invitation (just wtf IS the moghouse ID really?)
3. Update fallbacks for invalid zone data. Fixes longstanding issue where you wind up in the unused tech demo zone. (because this is NOT a residential zone.)
4. Zone ID are more than 8 bits. Oopsies on that uint8..

### CODE REVIEW

File: src/map/packet_system.cpp [hunk 1]
```diff
@@ -317,13 +318,16 @@ void SmallPacket0x00A(map_session_data_t* const PSession, CCharEntity* const PCh
...
     {
+        // TODO: this should only happen ONCE, instead of spamming the log dozens of times..
         ShowWarning("Client cannot receive packet or key is invalid: %s, Zone: %s (%i)",
             PChar->GetName(), PChar->loc.zone->GetName(), PChar->loc.zone->GetID());
 
         // Write a sane location for them
-        auto newZone = PChar->loc.prevzone ? PChar->loc.prevzone : (uint8)ZONE_VALKURM_DUNES;
-        PChar->loc.destination = newZone;
-        sql->Async("UPDATE chars SET pos_zone = %u WHERE charid = %u", newZone, PChar->id);
+        // TODO: work out how to drop plaer in moghouse that exits them to the zone they were in before this happened, like we used to.
```
Comment (Xaver-DaRed) [inline src/map/packet_system.cpp]: Typo in "player"

**Review by Xaver-DaRed — APPROVED** [review]

File: src/map/packet_system.cpp (hunk 1 above)
Comment (TeoTwawki) [reply src/map/packet_system.cpp]: \>.<

**Comment by zach2good** [comment]
```
Replace limited version of up command with the one I wrote for LegionDark _5 years ago_ and add its matching down command
```

This sort of commentary has no place in the commit history. What are you trying to achieve here?

**Comment by TeoTwawki** [comment]
we can rebase/squash the message out if its bothering you.

**Comment by TeoTwawki** [comment]
but we need to talk if this is your reaction to it

**Review by claywar — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

**Pull request merged**